spyder_env_manager/spyder/widgets/packages_table.py

```diff
@@ -16,51 +16,41 @@
 from qtpy.compat import to_qvariant
 from qtpy.QtCore import QAbstractTableModel, QModelIndex, Qt, Signal
 from qtpy.QtGui import QColor
-from qtpy.QtWidgets import QAbstractItemView, QLabel, QMenu, QTableView
+from qtpy.QtWidgets import QAbstractItemView, QTableView
 
 # Spyder and local imports
 from spyder.api.translations import get_translation
+from spyder.api.widgets.mixins import SpyderWidgetMixin
 from spyder.config.fonts import DEFAULT_SMALL_DELTA
 from spyder.config.gui import get_font
 from spyder.utils.palette import SpyderPalette
-from spyder.utils.qthelpers import add_actions, create_action
+
 
 # Localization
 _ = get_translation("spyder")
 
 
-PACKAGE, DESCRIPTION, VERSION = [0, 1, 2]
+# Column constants
+NAME, VERSION, DESCRIPTION = [0, 1, 2]
 
 
-class EnvironmentPackagesModel(QAbstractTableModel):
-    def __init__(self, parent, text_color=None, text_color_highlight=None):
-        QAbstractTableModel.__init__(self)
-        self._parent = parent
+class EnvironmentPackagesActions:
+    # Actions available for a package (from the context menu)
```

Comment:
```suggestion
    """
    Actions available for a package from the `EnvironmentPackagesTable`
    context menu.
    """

```

This can be a docstring.